test/zdtm/transition/fifo_loop.c

```diff
@@ -39,6 +39,7 @@ int main(int argc, char **argv)
 	int i;
 	uint8_t buf[0x100000];
```

Comment:
```
$ ag 0x100000
...
test/zdtm/transition/fifo_dyn.c
31:	uint8_t buf[0x100000];

test/zdtm/transition/fifo_loop.c
40:	uint8_t buf[0x100000];

test/zdtm/transition/pipe_loop00.c
34:	uint8_t buf[0x100000];

test/zdtm/transition/socket_loop00.c
35:	uint8_t buf[0x100000];
```
The default pipe buffer size is 0x100000 == 1048576
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/fs/pipe.c?id=HEAD#n37

Why not set the pipe's buffer size instead? At least it may save a few clock cycles every time we run the tests.

Reply:
This is a good idea. Thanks!